package.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "everydayhero <edh-dev@everydayhero.com>",
   "name": "edh-widgets",
-  "version": "3.8.5",
+  "version": "3.8.6",
```

Comment:
I'd prefer we do this on master using the `npm version` tool. No biggy, just means you can't now, when it lands on master you have to: make a tag, and make a new version commit (so the auto-deploy runs). `npm version` does all three things for you: updates package.json, makes a version commit, adds a tag.